backend/routes/edittrip.js

```diff
@@ -0,0 +1,14 @@
+const validation = require('../validation.js');
+const editTripPostHandler = require('../handlers/edittrippsot.js');
+const editTripPost = {
+  method:'POST',
```

Comment:
If you are editing/updating the post here, it should be a `PUT` request and the path should just be `/post`. Do not put verbs in your path. That is what the HTTP methods are for.